lib/helpers/check-attributes.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const calculateLocationDisplay = require('./calculate-location-display');
+const supportsAttribute = require('./supports-attribute');
+
+module.exports = function checkAttributes(node, supported) {
+  let sourceReference = calculateLocationDisplay(null, node.loc.start);
+
+  if (node.modifiers.length > 0) {
+    throw new Error(
```

Comment:
Mind swapping this to `silent-error`? The stack trace of this particular error will be super useless...

Reply:
Done. Also for the other place where we throw an error...